PowerEditor/src/NppIO.cpp

```diff
@@ -884,6 +895,10 @@ bool Notepad_plus::fileCloseAllButCurrent()
 			{
 					return false;
 			}
+			else if (res == IDNO && isLeftOrRightShiftPressed())

```

Comment:
Nitpick: Seems like this code and possibly surrounding code (I can't see all of it) belongs in a reusable private method or something.